src/main/kotlin/no/nav/faktureringskomponenten/controller/dto/FakturaResponseDto.kt

```diff
@@ -24,5 +28,7 @@ data class FakturaResponseDto(
     @Schema(description = "Sluttdato for perioden")
     val periodeTil: LocalDate,
 
-    val fakturaMottat: List<FakturaTilbakemeldingResponseDto>
+    val eksternFakturaStatus: List<FakturaTilbakemeldingResponseDto>,
+
+    val nyesteFakturaStatus: FakturaTilbakemeldingResponseDto? = null
```

Comment:
Ville `nyesteFakturaStatus` ikke være i listen `eksternFakturaStatus`?

Reply:
Jo, men istedenfor å sortere dette i frontend, tar jeg det her tenkte jeg er best

Reply:
Lurer på hvorfor du tenker dette er best.
Man pleier å gå for dataminimering ifm. API-design, så dette er ikke veldig intuitivt.
Listen kan ev. sorteres for frontend i stedet for å duplisere data.

Reply:
Jeg tok utgangspunkt i API hvor i fagsak henter vi sisteAvsluttet behandling osv. Jeg synes det var greit, men jeg kan endre til å sortere annet sted

Reply:
Skjønner. En fagsak kan eie flere behandlinger og gi siste behandling osv. 
Her er det ikke synkrone kall, men kall over nettverk. Så jeg tror det ville være forvirrende å bruke samme mønster.